projects/plugins/jetpack/extensions/blocks/map/map.php

```diff
@@ -51,7 +51,7 @@ function wpcom_load_event( $access_token_source ) {
 	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
 		jetpack_require_lib( 'tracks/client' );
 		tracks_record_event( wp_get_current_user(), $event_name );
-	} elseif ( jetpack_is_atomic_site() && Jetpack::is_active() ) {
+	} elseif ( jetpack_is_atomic_site() && Jetpack::connection()->is_user_connected() ) {
```

Comment:
I'm not sure about this one.

Are we only tracking when the user is present? If the idea is to track visits by other users, we can't use this.

I would go on the safe side here and use `is_connection_ready`

Reply:
I did this also to be on the safe side because of `jetpack_is_atomic_site` :)

I guess we can handle Atomic in `is_connection_ready` at a later stage.